### PR TITLE
add groupedgemm solutions for F8H NN

### PR DESCRIPTION
**Pull request by aazz44ss**

### CODE REVIEW

File: library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx941/FreeSize/aquavanjaram_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs.yaml [hunk 1]
```diff
@@ -117834,6 +117834,5433 @@
...
+    AssertFree0ElementMultiple: 4
+    AssertFree1ElementMultiple: 1
+    AssertSummationElementMultiple: 1
+    AssignedDerivedParameters: true
+    AssignedProblemIndependentDerivedParameters: true
+    BufferLoad: true
+    BufferStore: true
+    CUCount: null
+    ClusterLocalRead: 1
+    CodeObjectVersion: V3
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx941/FreeSize/aquavanjaram_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs.yaml:117850]: should be revised as "CodeObjectVersion: default"

Comment (aazz44ss) [reply library/src/amd_detail/rocblaslt/src/Tensile/Logic/asm_full/aquavanjaram/gfx941/FreeSize/aquavanjaram_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs.yaml:117850]: done

File: tensilelite/Tensile/CustomKernels/Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941.s [hunk 2]
```diff
@@ -0,0 +1,9297 @@
...
+   WaveSeparateGlobalReadB: 1
+   GlobalReadVectorWidthA: 4
+   GlobalReadVectorWidthB: 2
+   AssertFree0ElementMultiple: 4
+   AssertSummationElementMultiple: 1
+   WorkGroupMappingXCC: 1
+   NoReject: 1
+amdhsa.version:
+  - 1
+  - 0
```
Comment (jichangjichang) [inline tensilelite/Tensile/CustomKernels/Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941.s]: " - 0" should be replaced with "- 1" for CodeObjectV4

Comment (aazz44ss) [reply tensilelite/Tensile/CustomKernels/Custom_Cijk_Ailk_Bljk_F8H_HHS_BH_Bias_GG_AS_SAB_SAV_UserArgs_shortname0_gfx941.s]: done

**Review by jichangjichang — APPROVED** [review]
lgtm

**Review by KKyang — APPROVED** [review]

**Pull request merged**